.github/workflows/respond-to-pr.yaml

```diff
@@ -0,0 +1,39 @@
+name: 'Add Comment To New PR'
+on:
+  pull_request:
+    types: created
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  new_pr_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GrantBirki/comment@v2.1.0
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Thanks for raising this pull request!
+
+            The packages within this repository are maintained by a small team of volunteer Community Maintainers, in their own time. Their time, like yours is important. Please help them by not wasting it.
+
+            This is a community repository where the expectation is that everybody will get involved. Thank you for being one of those people who want to help out by raising a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). There are a couple of steps beyond this:
+
+              1. Please ensure that any automated tests pass. If they do not, the Community Maintainers Team will be unlikely to pick up the pull request until it does. Occasionally the automated tests fail and can be ignored. If you _really_ believe that is the case here, please leave a comment indicating this, and _why_.
+              1. If the automated tests pass, a member of the Community Maintainers Team will pick up the pull request and work through it to ensure it follows the [CONTRIBUTING](CONTRIBUTING.md) and [COMMITTERS](COMMITTERS.md) documentation. Please be patient. The team are volunteers.
+              1. Once the pull request has been picked up, the team may merge it, or ask for changes. They may also simply make the changes themselves if they have the time to do so. The goal is to get your pull request merged as quickly as possible. But time is limited, and they do the best they can.
+
+            A few dos and don'ts:
```

Comment:
I'd suggest mentioning verifying the changes in the Chocolatey Test Environment as a Do, because there have been a number of PRs recently that didn't have that box checked.

Reply:
Good idea. I've added that just now.